src/EPR.PRN.Backend.Obligation/Constants/ObligationConstants.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EPR.PRN.Backend.Obligation.Constants
+{
+    public static class ObligationConstants
+    {
+        public static class Statuses
+        {
+            public const string NoDataYet = "NO DATA YET";
+            public const string Met = "MET";
+            public const string NoMet = "NO MET";
```

Comment:
My front-end code expects this values to be "NoDataYet", "NotMet" and "Met". Same like material type (PascalCase). Which then i use to decide the status for "Totals" row.
public enum ObligationStatus
{
    NoDataYet
    NotMet,
    Met
}

Reply:
sure will change